src/Halogen/Svg/Core.purs

```diff
@@ -1,21 +1,34 @@
-module Halogen.Svg.Core where
+module Halogen.Svg.Core
+  ( ns
+  , element
+  , attr
+  ) where
 -- Like Halogen.HTML.Core
 
 import Prelude
-import Data.Maybe (Maybe(..))
-import Halogen.HTML.Core (HTML, Prop(Attribute), Namespace(Namespace), AttrName(AttrName))
+import Data.Maybe (Maybe(Just, Nothing))
+import Halogen.HTML.Core as H
 import Halogen.VDom (ElemName, VDom(Elem))
 import Unsafe.Coerce (unsafeCoerce)
 
-ns :: Maybe Namespace
-ns = Just $ Namespace "http://www.w3.org/2000/svg"
+ns :: Maybe H.Namespace
+ns = Just $ H.Namespace "http://www.w3.org/2000/svg"
 
-element :: forall p i. ElemName -> Array (Prop i) -> Array (HTML p i) -> HTML p i
+element :: forall p i.
+  ElemName -> Array (H.Prop i) -> Array (H.HTML p i) -> H.HTML p i
 element = coe (\name props children -> Elem ns name props children)
   where
-    coe :: (ElemName -> Array (Prop i) -> Array (VDom (Array (Prop i)) p) -> VDom (Array (Prop i)) p)
-        -> ElemName -> Array (Prop i) -> Array (HTML p i) -> HTML p i
+    coe :: CoerceFrom p i -> CoerceTo p i
```

Comment:
Please remove these type synonyms. I used to do this as well because I thought it made code easier to read. However, I think that's better done with comments. I don't want to accidentally export these type aliases later or something like that.

Reply:
Fixed via 2f14b2b9dd084c243083d80510efa947eb57ce88